### PR TITLE
feat(base-cluster): add kdave for deprecated CRD metrics

### PR DESCRIPTION
**Pull request by cwrau**

### CODE REVIEW

File: charts/base-cluster/values.yaml [hunk 1]
```diff
@@ -172,6 +172,14 @@ global:
       charts:
         opentelemetry-collector: 0.x.x
       condition: "{{ and .Values.monitoring.tracing.enabled .Values.monitoring.prometheus.enabled }}"
+    kdave:
+      url: https://github.com/amelbakry/kdave-chart
+      charts:
+        kdave:
+          commit: 47a46848222969d60bae87b4650614a9b7a85d00
```
[the remark this answers is not in the document]
Comment (cwrau) [reply charts/base-cluster/values.yaml:179]: We need *something* to pin to, and they don't have tags 🤷

And yes, it's still usable

[the remark this answers is not in the document]
Comment (marvinWolff) [reply charts/base-cluster/values.yaml:179]: Alright. Then go for it

**Review by marvinWolff — DISMISSED** [review]

File: charts/base-cluster/templates/monitoring/kdave/kdave.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{{- if and .Values.monitoring.prometheus.enabled .Values.monitoring.kdave.enabled -}}
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
```
Comment (tasches) [inline charts/base-cluster/templates/monitoring/kdave/kdave.yaml:2]:
```suggestion
apiVersion: helm.toolkit.fluxcd.io/v2
```

File: charts/base-cluster/templates/global/helmRepositories.yaml [hunk 3]
```diff
@@ -17,5 +18,30 @@ spec:
   type: oci
   {{- end }}
 ---
-{{ end -}}
-{{- end }}
+    {{ else -}}
+      {{- range $chartName, $chartConfig := $config.charts -}}
+apiVersion: source.toolkit.fluxcd.io/v1beta2
```
Comment (tasches) [inline charts/base-cluster/templates/global/helmRepositories.yaml:23]:
```suggestion
apiVersion: source.toolkit.fluxcd.io/v1
```

**Comment by marvinWolff** [comment]
I've fixed the missing image for kdave

**Review by tasches — APPROVED** [review]

**Pull request merged**